lumen/base.py

```diff
@@ -363,10 +363,19 @@ def _get_type(cls, component_type, spec=None):
             )
         if '.' in component_type:
             return resolve_module_reference(component_type, base_type)
+
         try:
-            __import__(f'lumen.{base_type.__name__.lower()}s.{component_type}')
-        except Exception:
-            pass
+            import_name = f'lumen.{base_type.__name__.lower()}s.{component_type}'
+            __import__(import_name)
+        except ImportError as e:
+            if e.name != import_name:
+                msg = (
+                    f"In order to use the {base_type.__name__.lower()} "
+                    f"component '{component_type}', the '{e.name}' package "
+                    "must be installed. It can be installed with:"
+                    f"\n  conda install {e.name}"
```

Comment:
It's a bit weird to just recommend `conda install`. I'd either not recommend anything, or recommend both `pip install` and `conda install`.

Reply:
The inspiration was taken from the error message in hvplot, but I can remove the recommendation.

https://github.com/holoviz/hvplot/blob/master/hvplot/converter.py#L1246-L1249

Reply:
Ah yes but that's specifically for datashader in hvPlot. While this error is for any package If I'm not mistaken.